cache_images/fedora_packaging.sh

```diff
@@ -206,6 +206,29 @@ DOWNLOAD_PACKAGES=(\
 msg "Installing general build/test dependencies"
 bigto $SUDO dnf install -y $EXARG "${INSTALL_PACKAGES[@]}"
 
+# FIXME: 2023-09-26: emergency upgrade to fix chmod-symlink bug
+if [[ $(date +%Y%m) -gt 202309 ]]; then
+    echo ""
+    echo "FATAL FATAL FATAL: REMOVE THIS TEMPORARY WORKAROUND"
+    echo
+    exit 1
+fi
+source /etc/os-release
+# shellcheck disable=SC2154
+if [[ $VERSION_ID -ge 38 ]]; then
```

Comment:
This is going to override on rawhide also.  I'm assuming that's desired.

---

For future ref.: The distro name and release are defined by both `lib.sh` (here) and (elsewhere) in the common automation library.  Something like:

```
OS_RELEASE_VER="$(source /etc/os-release; echo $VERSION_ID | tr -d '.')"
OS_RELEASE_ID="$(source /etc/os-release; echo $ID)"
OS_REL_VER="$OS_RELEASE_ID-$OS_RELEASE_VER"
```

In this script's context, there's also `$PACKER_BUILD_NAME` which can be used to distinguish Fedora vs Rawhide.